web/src/buddy_list.ts

```diff
@@ -89,6 +101,7 @@ function get_render_data(): BuddyListRenderData {
     const other_users_count = people.get_active_human_count() - total_human_subscribers_count;
     const hide_headers = should_hide_headers(current_sub, pm_ids_set);
     const get_all_participant_ids = buddy_data.get_conversation_participants_callback();
+    const non_participant_subscribers_count = get_non_participant_subscribers_count(current_sub);
```

Comment:
Unfortunately I don't think we can actually do this, which I didn't realize until yesterday, so sorry for misdirecting you. 

Reasoning: At the time `get_render_data` is called, the participants aren't loaded yet. This is why `get_all_participant_ids` is a function to be called later, instead of just an array of ids.

I'd recommend making a function `non_participant_subscribers_count` below `subscribed_participant_ids` that you call each time you need this value.